Documentation/network/kubernetes/compatibility.rst

```diff
@@ -17,12 +17,12 @@ with Cilium. Older Kubernetes versions not listed in this table do not have
 Cilium support. Newer Kubernetes versions, while not listed, will depend on the
 backward compatibility offered by Kubernetes.
 
-+------------------------------------------------------------+---------------------------+----------------------------------+
-| k8s Version                                                | k8s NetworkPolicy API     | CiliumNetworkPolicy              |
-+------------------------------------------------------------+---------------------------+----------------------------------+
-|                                                            |                           | ``cilium.io/v2`` has a           |
-| 1.16, 1.17, 1.18, 1.19, 1.20, 1.21, 1.22, 1.23, 1.24, 1.25 | * `networking.k8s.io/v1`_ | :term:`CustomResourceDefinition` |
-+------------------------------------------------------------+---------------------------+----------------------------------+
++------------------------------------------------------------------+---------------------------+----------------------------------+
+| k8s Version                                                      | k8s NetworkPolicy API     | CiliumNetworkPolicy              |
++------------------------------------------------------------------+---------------------------+----------------------------------+
+|                                                                  |                           | ``cilium.io/v2`` has a           |
+| 1.16, 1.17, 1.18, 1.19, 1.20, 1.21, 1.22, 1.23, 1.24, 1.25, 1.26 | * `networking.k8s.io/v1`_ | :term:`CustomResourceDefinition` |
```

Comment:
When are we dropping support for old versions, please please pretty please? 🥺 @aanm @joestringer

Reply:
I'm on the process of writing what concrete steps we will do before 1.13.

Reply:
@nbusseneau I'm on the process of writing what concrete steps we will do before 1.13.